.gitignore

```diff
@@ -2,6 +2,7 @@ data/
 
 .idea/
 *.db
+package-lock.json
```

Comment:
package-lock.json was added to the .gitignore but not actually removed. Pick one...

Reply:
Updated!